### PR TITLE
API-6111

### PR DESCRIPTION
**Pull request by syed-ali-hmrc**

### CODE REVIEW

File: .scalafix.conf [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  groupedImports = Merge
+  groups = [
+    "re:(javax?|scala)\\."
+    "---"
+    "*"
+    "---"
+    "play"
+    "uk.gov.hmrc."
+    "---"
+    "uk.gov.hmrc.apiplatformjobs."
```
Comment (petekirby-ee) [inline .scalafix.conf:19]: Need to change this package name to the correct one for this repo

**Comment by anjumabbas5** [comment]
Copyright 2021 found in 4 files

File: scalastyle-config.xml [hunk 2]
```diff
@@ -10,12 +10,7 @@
     <check level="warning" class="org.scalastyle.scalariform.SpacesAfterPlusChecker" enabled="true"/>
     <check level="warning" class="org.scalastyle.file.WhitespaceEndOfLineChecker" enabled="true"/>
     <check level="warning" class="org.scalastyle.scalariform.SpacesBeforePlusChecker" enabled="true"/>
-    <check level="warning" class="org.scalastyle.file.FileLineLengthChecker" enabled="true">
```
Comment (johnsgp) [inline scalastyle-config.xml:13]: I think we should rather disable than delete this. We should have a line length check of perhaps a bit longer.

**Comment by syed-ali-hmrc** [comment]
> Copyright 2021 found in 4 files

**Review by petekirby-ee — APPROVED** [review]

**Pull request merged**